src/components/icons/platform/index.jsx

```diff
@@ -130,12 +130,28 @@ const Link = styled.div`
   color: ${({ color }) => color};
   transition: border-color 0.25s ease-out, background-color 0.25s ease-out, color 0.25s ease-out;
 
-  width: ${({ isPreviewMobile, size }) =>
-    isPreviewMobile ? ShapeSize.Mobile[size] : `calc(${ShapeSize.Desktop[size]} - 15px)`};
-  height: ${({ isPreviewMobile, size }) =>
-    isPreviewMobile ? ShapeSize.Mobile[size] : `calc(${ShapeSize.Desktop[size]} - 15px)`};
-  margin: ${({ isPreviewMobile, size, margin }) =>
-    (isPreviewMobile && ((size === 'L' && '2px') || '5px')) || margin || '0.5rem 0.35rem'};
+  width: ${({ isSidePreview, isPreviewMobile, size }) =>
+    isPreviewMobile
+      ? `calc(${ShapeSize.Mobile[size]} ${(isSidePreview && '- 4px') || ' - 0px'});`
+      : `calc(${ShapeSize.Desktop[size]} ${
+          (isSidePreview &&
+            ` - ${parseInt(ShapeSize.Desktop[size]) - parseInt(ShapeSize.Mobile[size])}px`) ||
```

Comment:
parseInt should not be necessary. size is always defined and the ShapeSize is a constant